src/model/permissions.rs

```diff
@@ -810,6 +812,13 @@ impl Permissions {
     pub const fn send_polls(self) -> bool {
         self.contains(Self::SEND_POLLS)
     }
+    /// Shorthand for checking that the set of permissions contains the [Use External Apps] permission.
+    ///
```

Comment:
```suggestion

    /// Shorthand for checking that the set of permissions contains the [Use External Apps]
    /// permission.
```

Formatting change to pass CI.